irohad/model/commands/add_asset_quantity.hpp

```diff
@@ -33,18 +32,18 @@ namespace iroha {
       /**
        * Account where to add assets
        */
-      std::string account_id{};
+      std::string account_id;
```

Comment:
What is the reason behind removal of initialization?

Reply:
I see only one advantage of using that initialization -- it saves you some code and time when you have to initialize many vars in many constructors.

However, one disadvantage is that it decreases overall coverage -- std::string may not be constructed and this line is marked as partial.